utils/lint_check.sh

```diff
@@ -1,29 +1,35 @@
 #!/bin/bash
 
-topdir="`dirname $(dirname $0)`"
+WARN="\033[33;1m"
+RST="\033[0m"
 
+topdir="`dirname $0`/.."
+
+echo "Running 'flake8'..."
 flake8 .
+echo "Running 'pydocstyle'..."
 pydocstyle .
+echo "Running 'pylint'..."
 pylint plugins
 
-ANSIBLE_LIBRARY=${ANSIBLE_LIBRARY:-"${topdir}/plugins/modules"}
+ANSIBLE_LIBRARY="${ANSIBLE_LIBRARY:-${topdir}/plugins/modules}"
 ANSIBLE_MODULE_UTILS=${ANSIBLE_MODULE_UTILS:-"${topdir}/plugins/module_utils"}
 
 export ANSIBLE_LIBRARY ANSIBLE_MODULE_UTILS
```

Comment:
ANSIBLE_DOC_FRAGMENT_PLUGINS needs to be set also.

Reply:
Fixed.